hyperspy/roi.py

```diff
@@ -787,8 +790,10 @@ def parameters(self):
         Dictionary containing the parameters that define the current ROI.
 
         For a ``Point2DROI``, the dictionary contains:
+
         - ``x`` : float
         The x-coordinate of the point.
+
         - ``y`` : float
         The y-coordinate of the point.
```

Comment:
I would suggest to make the lists more concise, as example:
```suggestion
        - ``x`` : float – The x-coordinate of the point.
        - ``y`` : float – The y-coordinate of the point.
```